src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py

```diff
@@ -54,3 +55,34 @@ def test_plot_save_saves_plot(mock_save):
     name = "test_plot.png"
     cl.save(name)
     mock_save.assert_called_with(name)
+
+
+@mock.patch(
```

Comment:
Remove this mock!

Reply:
Only mock was to be removed, not the test !

Reply:
Done